whd/dispatcher.py

```diff
@@ -139,6 +143,77 @@ def _pipeline_definition_changed(self, push_event):
             return True
         return False
 
+    def determine_affected_pipelines(self, push_event):
+        '''yield the triple (pipeline name, target team, effective definition) for each concourse
```

Comment:
I would prefer a dataclass tbh. It is very little boilerplate, but helps maintainability a lot